types/tests/controllers/doughnut_meta_total.ts

```diff
@@ -0,0 +1,16 @@
+import { Chart, ChartMeta, Element } from '../../index.esm';
+
+const chart = new Chart('id', {
+  type: 'doughnut',
+  data: {
+    labels: [],
+    datasets: [{
+      data: [],
+    }]
+  },
+});
+
+// A cast is required because the exact type of ChartMeta will vary with
+// mixed charts
+const meta = <ChartMeta<Element, Element, 'doughnut'>>chart.getDatasetMeta(0);
```

Comment:
I'm not sure if the casting is acceptable here. It would be a lot simpler to cast if `TType` was the first parameter, but I wasn't sure that would be considered a breaking change or not

Reply:
Ideally it should not need to be casted, because its not a mixed chart. I don't know if there is a way to type it so it would return the correct meta for the given dataset index.